.changes/next-release/Feature-b473d1cf-ceff-4d40-9024-29d045e4dbee.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Added Hyperlink to toolkits telemetry documentation"
```

Description is missing context. Customers reading this description won't know that it is referring to the Toolkit telemetry setting.
```suggestion
	"description": "Telemetry setting description includes a details link"
```